integration/container/checkpoint_test.go

```diff
@@ -47,6 +48,11 @@ func TestCheckpoint(t *testing.T) {
 
 	defer setupTest(t)()
 
+	cmd := exec.Command("criu", "check")
```

Comment:
`criu check --all` ?

Reply:
I think that for this test just checking the required features is enough, checking all would require more effort like enabling uffd and specific tmpfs configurations.
I would say, let's merge this and then we can add more specific test cases.